tools/set_sk_env_vars.sh

```diff
@@ -7,9 +7,13 @@ if [ "`uname -o 2>>/dev/null`" = "Msys" ]; then
         exit 1
     elif [ "$MSYSTEM" = "MINGW64" ]; then
         export SK_OS="win64"
+    elif [ "$MSYSTEM" = "CLANG64" ]; then
+        export SK_OS="win64"
+    # elif [ "$MSYSTEM" = "CLANGARM64" ]; then
+    #     export SK_OS="winARM"
     else
         echo "Unable to detect Windows version..."
-        echo "Please run in MinGW64 terminal"
+        # echo "Please run in MinGW64 terminal"
```

Comment:
Could we use something to detect CPU architecture here: maybe `uname -p`.  Then we can indicate the terminal to use. Is there any way we can launch the terminal from here ourselves...

Reply:
I just tried testing the installation on my Windows ARM VM on my macbook and it is all working correctly (tested C++ and python so far) with the regular MINGW64 terminal... 

I will need to check if there is some x64 emulation happening somewhere in VMWare Fusion, but for now I have just reverted things back to using MINGW64 (with the other parts commented for future use if/when needed) so that we can work on the windows library building in a future PR and get the python fix in sooner.